packages/web-components/src/utils/chart-helpers.ts

```diff
@@ -60,3 +60,147 @@ const validateChartProps = (obj: any) => {
     });
   }
 };
+
+export const DataVizPalette = {
+  color1: 'qualitative.1',
+  color2: 'qualitative.2',
+  color3: 'qualitative.3',
+  color4: 'qualitative.4',
+  color5: 'qualitative.5',
+  color6: 'qualitative.6',
+  color7: 'qualitative.7',
+  color8: 'qualitative.8',
+  color9: 'qualitative.9',
+  color10: 'qualitative.10',
+  color11: 'qualitative.11',
+  color12: 'qualitative.12',
+  color13: 'qualitative.13',
+  color14: 'qualitative.14',
+  color15: 'qualitative.15',
+  color16: 'qualitative.16',
+  color17: 'qualitative.17',
+  color18: 'qualitative.18',
+  color19: 'qualitative.19',
+  color20: 'qualitative.20',
+  color21: 'qualitative.21',
+  color22: 'qualitative.22',
+  color23: 'qualitative.23',
+  color24: 'qualitative.24',
+  color25: 'qualitative.25',
+  color26: 'qualitative.26',
+  color27: 'qualitative.27',
+  color28: 'qualitative.28',
+  color29: 'qualitative.29',
+  color30: 'qualitative.30',
+  color31: 'qualitative.31',
+  color32: 'qualitative.32',
+  color33: 'qualitative.33',
+  color34: 'qualitative.34',
+  color35: 'qualitative.35',
+  color36: 'qualitative.36',
+  color37: 'qualitative.37',
+  color38: 'qualitative.38',
+  color39: 'qualitative.39',
+  color40: 'qualitative.40',
+  info: 'semantic.info',
+  disabled: 'semantic.disabled',
+  highError: 'semantic.highError',
+  error: 'semantic.error',
+  warning: 'semantic.warning',
+  success: 'semantic.success',
+  highSuccess: 'semantic.highSuccess',
+};
+
+/**
+ * Key: Color code.
+ * Value:
+ * Index 0 - Default color / Color for light theme,
+ * Index 1 - Color for dark theme
+ */
+type Palette = { [key: string]: string[] };
+
+const QualitativePalette: Palette = {
+  '1': ['#637cef'], // [cornflower.tint10],
+  '2': ['#e3008c'], // [hotPink.primary],
+  '3': ['#2aa0a4'], // [teal.tint20],
+  '4': ['#9373c0'], // [orchid.tint10],
+  '5': ['#13a10e'], // [lightGreen.primary],
+  '6': ['#3a96dd'], // [lightBlue.primary],
+  '7': ['#ca5010'], // [pumpkin.primary],
+  '8': ['#57811b'], // [lime.shade20],
+  '9': ['#b146c2'], // [lilac.primary],
+  '10': ['#ae8c00'], // [gold.shade10],
+  '11': ['#3c51b4', '#93a4f4'], // [cornflower.shade20, cornflower.tint30],
+  '12': ['#ad006a', '#ee5fb7'], // [hotPink.shade20, hotPink.tint30],
+  '13': ['#026467', '#4cb4b7'], // [teal.shade20, teal.tint30],
```

Comment:
remove 11-20 and 30-40. We will add them back once we have a solution for dark mode.

Reply:
done